packages/core/src/browser/keybinding.ts

```diff
@@ -344,12 +348,15 @@ export class KeybindingRegistry {
 
             matches.full = matches.full.filter(
                 binding => this.getKeybindingCollisions(result.full, binding).full.length === 0);
+            matches.normalized = matches.normalized.filter(
+                binding => this.getKeybindingCollisions(result.normalized, binding).normalized.length === 0);
```

Comment:
@marechal-p it is picked up here

Reply:
Ok, but in this change I don't see anything consuming the `matches.normalize` list?

Reply:
It gets merged into the result.